### PR TITLE
df-1341-improve-documentation-for-repo-configuration

### PR DESCRIPTION
**Pull request by Dmitriy-Kulagin**

### CODE REVIEW

File: repo.md [hunk 1]
```diff
@@ -17,12 +20,15 @@ See the results of the data checks in the action logs.
...
 - We have created a repository for you and you will need to fork it.
 - Go to _Settings → Secrets → Action_ of your forked repository
-- Add the created __Personal access token__ by calling it `ACTION_TOKEN` here
+- Press __New repository secret__ and add created __Personal access token__ with name `ACTION_TOKEN` here
+![git_add_secret]
 - Now go to _Actions → General → Action permissions_
 - Check the [Allow all actions and reusable workflows][gh_docs_actions] box here
+![git_allow_actions]
 - Click the _Actions_ tab
 - Disable all workflows and only enable __Check data and create pr__
```
Comment (DORE145) [inline repo.md:33]: Please add screenshots here as well with a special note that some of them could be hidden and there is a "Show hidden workflows" button

Comment (Dmitriy-Kulagin) [reply repo.md:34]: The additional screenshot is added

File: repo.md [hunk 2]
```diff
@@ -16,41 +14,53 @@ See the results of the data checks in the action logs.
...
+symbol_info          # Your JSON-file with symbol information
 README.md
 ```
 
 ## Add data files
 
-- Upload CSV data files to the `data/repo_name` directory
-- Upload a JSON character description file to the `symbol_info` directory
+- Upload CSV data files to the `data/repo_name` directory.
+- Upload a JSON character description file to the `symbol_info` directory.
```
Comment (osal) [inline repo.md:54]: @anachuprina, character -> symbol

File: repo.md [hunk 3]
```diff
@@ -48,17 +50,17 @@ README.md
...
 
 ## Check the data upload
 
-Data is regularly checked and loaded into TradingView storage by the __Check data and create pr__ action. 
-The results of the data checks can be found in the action logs.
+The __Check data and create pr__ action regularly checks data and loads it into the TradingView storage.
```
Comment (KatPierce) [inline repo.md:58]: Экшн проверяет корректность данных или изменения? Или и то, и то?

**Review by KatPierce — APPROVED** [review]

**Pull request merged**